src/components/MetadataCollections/MetadataCollections.js

```diff
@@ -192,15 +203,53 @@ class MetadataCollections extends React.Component {
     />
   );
 
+  cacheFilter(activeFilters, searchValue) {
+    localStorage.setItem('fincConfigCollectionFilters', JSON.stringify(activeFilters));
+    localStorage.setItem('fincConfigCollectionSearchString', JSON.stringify(searchValue));
+  }
+
+  resetAll(getFilterHandlers, getSearchHandlers) {
+    localStorage.removeItem('fincConfigCollectionFilters');
+    localStorage.removeItem('fincConfigCollectionSearchString');
+
+    // reset the filter state to default filters
+    getFilterHandlers.state(defaultFilter.state);
+
+    // reset the search query
+    getSearchHandlers.state(defaultSearchString);
+
+    this.setState({
+      storedFilter: defaultFilter,
+      storedSearchString: defaultSearchString,
+    });
+
+    return (this.props.history.push(`${urls.collections()}?filters=${defaultFilter.string}`));
+  }
+
+  handleClearSearch(getSearchHandlers, onSubmitSearch, searchValue) {
```

Comment:
The function does not use the parameter _onSubmitSearch_. Can be removed.

Reply:
_onSubmitSearch_ is necessary, its in the return statement now